packages/util/src/constants.ts

```diff
@@ -67,6 +67,11 @@ export const KECCAK256_RLP = hexToBytes(KECCAK256_RLP_S)
 
 export const SHA256_NULL = sha256(new Uint8Array())
 
+/**
+ * SHA-256 hash of the RLP of an empty requests hash
+ */
+export const SHA256_EMPTY_RH = sha256(new Uint8Array([0, 1, 2]))
```

Comment:
This here would become part of the official API of the libraries and it would be hard "to get rid of it" if things change, so let's please not do this yet as long as specs are still changing and rather hardcode in blockchain.

Reply:
Hi Holger, good point, I did somewhat notice this but note that the "empty requests hash" will be gone starting of devnet-5 (see: https://github.com/ethereum/EIPs/pull/8989), where it is the empty sha256 hash and not this thing with encoded constants (so we can get rid of this constant!). But I agree it should not have been put in the util constants (note: this PR is not merged into master, it is merged in our devnet4 branch, so for now this is fine! :smile: )

Reply:
Should nevertheless not forget to remove this constant :smile: :+1:

Reply:
These things DO have some tendency to get forgotten, so I would still pledge to just give this a quick update *now* and we are done with it.

Reply:
Done in https://github.com/ethereumjs/ethereumjs-monorepo/pull/3706/commits/dc93f8f0cec8b0b025d3c393f1b336ee08349e73